### ISSUE TITLE
Add socket.io sample project

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/browser-template#82]
Idea per @gaand is to update `game-project-api` to use Rails 5 (and other changes in `rails-api-template`), make sure that it still supports `server-sent-events` -> eventSource, and if not we NEED to update this to allow for socket.io (before we can update `game-project-api`), otherwise it is a nice to have for when we get web sockets working in the rails template. This is a serious upgrade that would be great for devs trying to exceed.
